communication/accounts.go

```diff
@@ -0,0 +1,40 @@
+package communication
+
+import (
+	accountsv1 "notes-service/protorepo/noted/accounts/v1"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type AccountsServiceClient struct {
+	conn     *grpc.ClientConn
```

Comment:
Why do you store the `grpc.ClientConn` in conn ?
The connection is no longer usefull after the newAPIClient created.

Reply:
make it easier to close the connection at the end